assets/chezmoi.io/docs/reference/templates/github-functions/gitHubLatestRelease.md

```diff
@@ -1,12 +1,12 @@
-# `gitHubLatestRelease` *owner-repo*
+# `gitHubLatestRelease` *host-owner-repo*
```

Comment:
We may want to show this as *either `*[host-]owner-repo*` or `*owner-repo*|*host-howner-repo*`.

This applies to all `github*` functions.